src/pytest_plugins/filler/filler.py

```diff
@@ -265,6 +265,17 @@ def pytest_addoption(parser: pytest.Parser):
         default=False,
         help="Fill tests using existing pre-allocation groups (phase 2 only).",
     )
+    test_group.addoption(
+        "--gas-benchmark-values",
+        action="store",
+        dest="gas_benchmark_value",
+        type=str,
+        default=None,
```

Comment:
```suggestion
        default='5',
```
This would default to run the benchmark tests with 5 million gas for sanity checking for example, what do you think?

Reply:
> We need to be careful with non-benchmark tests, if a test is not a benchmark test it should be unaffected.
According to the issue description, non-benchmark tests shouldn’t be impacted, since this would change the gas limit.

If we set the default value of `--gas-benchmark-values` to "5", the non-benchmark ones would automatically apply a benchmark fixture with a 5M gas setting.

Based on my experiment, setting the default to "5" doesn’t apply when the flag is explicitly included without a value, like this:
```bash
uv run fill -v tests/benchmark/test_worst_blocks.py::test_block_full_data \
  --fork Prague \
  --gas-benchmark-values\
  --generate-pre-alloc-groups \
  --clean
# This will lead to error
```

However, when the flag is ignored the default value is applied, as seen in this command:
```bash
uv run fill -v tests/benchmark/test_worst_blocks.py::test_block_full_data \
  --fork Prague \
  --generate-pre-alloc-groups \
  --clean
# It will have fork_Prague-blockchain_test_engine_x_from_state_test-benchmark-gas-value_5M-zero_byte_False
```